appeals/api/src/server/endpoints/appeal-decision/__tests__/appeal-decision.test.js

```diff
@@ -191,7 +191,7 @@ describe('appeal decision routes', () => {
 						lpa_reference: '48269/APP/2021/1482',
 						site_address: '96 The Avenue, Leftfield, Maidstone, Kent, MD21 5XY, United Kingdom',
 						url: FRONT_OFFICE_URL,
-						decision_date: '4 June 2024'
+						decision_date: '7 June 2024'
```

Comment:
THis test is going to fail after the 17/06/2024 as the date is calculated as today -10 days

Reply:
Lets wait for the fix PR (https://github.com/Planning-Inspectorate/appeals-back-office/pull/228) to merge, we can then rebase this PR and hopefully merge both.